.github/workflows/zephyr.yml

```diff
@@ -16,16 +16,9 @@ jobs:
       matrix:
         # Using groups to avoid spamming the small results box with too
         # many lines. Pay attention to COMMAS.
-        IPC_platforms: [
-          # - IPC3 default
-          apl cnl,
-          icl jsl,
-          tgl tgl-h,
-          imx8 imx8x imx8m,
```

Comment:
Why was imx removed?

Reply:
is imx a zephyr platform?

Reply:
Yes, IMX is a Zephyr platform.

Reply:
Why is TGL removed?

Reply:
tgl is build only with ipc4 being native drivers overlay - thus using zephyr drivers right?

Reply:
Funny how TGL-H IPC3 was removed before TGL-H IPC4 compiles.